src/ChainRules.jl

```diff
@@ -10,6 +10,8 @@ using Random
 using RealDot: realdot
 using SparseArrays
 using Statistics
+using Distributed
+using DistributedArrays
```

Comment:
Distributed is a standard library but DistributedArrays is a package with these dependencies:

https://juliahub.com/ui/Packages/DistributedArrays/Gfdem/0.6.6?page=1

I don't know what others think about adding dependencies. What exactly is gained, what are the costs?